condition/bitmath.go

```diff
@@ -0,0 +1,103 @@
+package condition
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/brexhq/substation/config"
+	"github.com/brexhq/substation/internal/errors"
+)
+
+// bitmath evaluates data using bitwith math operations.
+//
+// This inspector supports the data and object handling patterns.
+type inspBitmath struct {
+	condition
+	Options inspBitmathOptions `json:"options"`
+}
+
+type inspBitmathOptions struct {
```

Comment:
docs: There are some copy/pastes from the strings inspector in here.

Reply:
Good catch, should be updated now! 👍